pkg/sql/opt/exec/execbuilder/testdata/inverted_index

```diff
@@ -273,33 +273,37 @@ filter     ·                    ·                     (a, b)  ·
 ·          spans                FULL SCAN             ·       ·
 
 
+# TODO(mgartner): It should not be required to force the index scan. It is
+# required until the statistics builder treats b->'a' = '"b"' similarly to the
+# containment operator, @>.
 query TTTTT
-EXPLAIN (VERBOSE) SELECT * from d where b->'a' = '"b"'
+EXPLAIN (VERBOSE) SELECT * from d@foo_inv where b->'a' = '"b"'
 ----
 ·           distribution         local                        ·       ·
 ·           vectorized           true                         ·       ·
 index join  ·                    ·                            (a, b)  ·
- │          estimated row count  110 (missing stats)          ·       ·
+ │          estimated row count  333 (missing stats)          ·       ·
  │          table                d@primary                    ·       ·
  │          key columns          a                            ·       ·
  └── scan   ·                    ·                            (a)     ·
 ·           estimated row count  110 (missing stats)          ·       ·
 ·           table                d@foo_inv                    ·       ·
 ·           spans                /"a"/"b"-/"a"/"b"/PrefixEnd  ·       ·
 
+# TODO(mgartner): Add support for building inverted index constraints for chained JSON
+# fetch operators.
```

Comment:
I created a ticket to track here: #55317

Reply:
I also created a ticket to track the `j->'a' @> '{"x": "y"}` case: https://github.com/cockroachdb/cockroach/issues/55318